doc/control-tower.md

```diff
@@ -0,0 +1,160 @@
+# Path to AWS Control Tower
+
```

Comment:
We should add a section to explain why we would go that way. I don't have a clear answer here but here might be few arguments: 
* Team size increasing with more segregated roles (Cloud admin, devs, ops ... etc.) hence need more than a git service to manage authN/authZ
* Compliancy (CT comes with a bunch of pre packaged guard rails to help reach compliancy with specific standards: https://aws.amazon.com/blogs/mt/extend-aws-control-tower-governance-using-aws-config-conformance-packs/)
* Move from Infra as Code for account creation and go with  web interface with [CT Account factory](https://docs.aws.amazon.com/controltower/latest/userguide/account-factory.html)
* Move from Infra as Code for Org structure and go with CT Web interface

Reply:
OK